screens/FavPokemon.tsx

```diff
@@ -38,7 +40,7 @@ export default function FavPokemon({ navigation }) {
         <View style={styles.propCard}>
           <PropRow
             left='Type:'
-            right={favPokemon.types.map(type => type['type']['name'])}
+            right={types.map(type => type['type']['name'])}
```

Comment:
I think it's better to keep `PropRow` type as `string` (right now it's string or `Array<string>`)). Could you create one string out of types?

Reply:
sure!